src/fmeval/model_runners/extractors/jumpstart_extractor.py

```diff
@@ -62,22 +69,39 @@ def __init__(
             lambda x: x.get(MODEL_ID, None) == jumpstart_model_id
         )
         util.require(model_manifest, f"Model {jumpstart_model_id} is not a valid JumpStart Model")
-        model_spec_key = self.get_jumpstart_sdk_spec(
+
+        model_spec = self.get_jumpstart_sdk_spec(
             model_manifest.get(SPEC_KEY, None),
             self._sagemaker_session.boto_region_name,
         )
-        util.require(
-            DEFAULT_PAYLOADS in model_spec_key, f"JumpStart Model: {jumpstart_model_id} is not supported at this time"
-        )
+        if DEFAULT_PAYLOADS not in model_spec:
+            # Model spec contains alt configs, which should
+            # be obtained through JumpStart util function.
+            logger.info(
+                "default_payloads not found as a top-level attribute of model spec"
+                "Searching for default_payloads in inference configs instead."
+            )
+            model_spec = verify_model_region_and_return_specs(
+                region=self._sagemaker_session.boto_region_name,
+                model_id=self._model_id,
+                version=self._model_version,
+                model_type=self._model_type,
+                scope=JumpStartScriptScope.INFERENCE,
+                sagemaker_session=self._sagemaker_session,
+            )
+            configs = model_spec.inference_configs  # type: ignore[attr-defined]
+            util.require(configs, f"JumpStart Model: {jumpstart_model_id} is not supported at this time")
+            default_payloads = configs.get_top_config_from_ranking().resolved_metadata_config[DEFAULT_PAYLOADS]
+        else:
+            # Continue to extract default payloads by manually parsing the spec json object.
+            # TODO: update this code when the `default_payloads` attribute of JumpStartModelSpecs
+            #  returns the full data, including fields like generated_text.
+            default_payloads = model_spec[DEFAULT_PAYLOADS]
```

Comment:
don't you also need to do `util.require(default_payloads, f"JumpStart Model: {jumpstart_model_id} is not supported at this time")`